core/templates/components/graph-services/graph-layout.service.spec.ts

```diff
@@ -950,56 +1022,62 @@ describe('Graph Layout Service', () => {
         id: 'State5',
         label: 'State5',
         height: 33.6,
-        width: 113.40000000000002,
+        width: 126,
         reachableFromEnd: false
       }
     };
-    // The expectedGraphBoundaries are calculated from the x0 and y0 values from
-    // nodeDataWithPositionValueInPixel, where leftEdge is
-    // minimum(nodeDataWithPositionValueInPixel[nodeId].x0 - BORDER_PADDING)
-    // of all nodes, and rightEdge is
-    // maximum(nodeDataWithPositionValueInPixel[nodeId].x0 + BORDER_PADDING +
-    // nodeDataWithPositionValueInPixel[nodeId].width) of all nodes. Similarly,
-    // bottomEdge and topEdge are calculated using y0 and node height.
-    let expectedGraphBoundaries1 = sgls.getGraphBoundaries(
+
+    let actualGraphBoundaries1 = sgls.getGraphBoundaries(
       nodeDataWithPositionValueInPixel1);
-    let expectedGraphBoundaries2 = sgls.getGraphBoundaries(
+    let actualGraphBoundaries2 = sgls.getGraphBoundaries(
       nodeDataWithPositionValueInPixel2);
-    let expectedGraphBoundaries3 = sgls.getGraphBoundaries(
+    let actualGraphBoundaries3 = sgls.getGraphBoundaries(
       nodeDataWithPositionValueInPixel3);
 
     // The width of graph calculated as difference b/w left and right edge.
-    let expectedWidth1 = expectedGraphBoundaries1.right -
-      expectedGraphBoundaries1.left;
-    let expectedWidth2 = expectedGraphBoundaries2.right -
-      expectedGraphBoundaries2.left;
-    let expectedWidth3 = expectedGraphBoundaries3.right -
-      expectedGraphBoundaries3.left;
-
-    // This is the maximum upperbound for graph witdh taking padding fraction
-    // into consideration.
-    let widthUpperBound = 548.6500000000001;
+    let actualWidth1 = actualGraphBoundaries1.right -
+      actualGraphBoundaries1.left;
+    let actualWidth2 = actualGraphBoundaries2.right -
+      actualGraphBoundaries2.left;
+    let actualWidth3 = actualGraphBoundaries3.right -
+      actualGraphBoundaries3.left;
+
+    // This is the maximum upperbound for graph width taking padding fraction
```

Comment:
Here and elsewhere: "upper bound" and "lower bound" are two words.

Reply:
Done